k8s/influxdb/values.yaml

```diff
@@ -0,0 +1,256 @@
+## influxdb image version
+## ref: https://hub.docker.com/r/library/influxdb/tags/
+image:
+  repository: "influxdb"
+  tag: "1.7.10-alpine"
+  pullPolicy: IfNotPresent
+  ## If specified, use these secrets to access the images
+  # pullSecrets:
+  #   - registry-secret
+
+
+serviceAccount:
+  create: true
+  name:
+  annotations: {}
+
+## Customize liveness, readiness and startup probes
+## ref: https://docs.influxdata.com/influxdb/v1.7/tools/api/#ping-http-endpoint
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+##
+livenessProbe: {}
+  # initialDelaySeconds: 30
+  # timeoutSeconds: 5
+  # scheme: HTTP
+
+readinessProbe: {}
+  # initialDelaySeconds: 5
+  # timeoutSeconds: 1
+  # scheme: HTTP
+
+securityContext: {}
+  # runAsUser: 999
+  # runAsGroup: 999
+
+startupProbe:
+  enabled: false
+  # failureThreshold: 6
+  # periodSeconds: 5
+  # scheme: HTTP
+
+## Specify a service type
+## NodePort is default
+## ref: http://kubernetes.io/docs/user-guide/services/
+##
+service:
+  ## Add annotations to service
+  # annotations: {}
+  type: ClusterIP
+
+## Persist data to a persistent volume
+##
+persistence:
+  enabled: false
```

Comment:
Persistence is disabled.

Reply:
Why? Shouldn't we enable it?

Reply:
It's simpler if data goes away in case this crashes. Also this is just for debug data right now.

The problem with `enabled: true` is that we then get an EBS volume, which might create additional issues when trying to attach/detach it.